astropy/io/ascii/docs.py

```diff
@@ -48,13 +50,13 @@
         List of names to include in output.
     exclude_names : list
         List of names to exclude from output (applied after ``include_names``)
-    fill_values : dict
+    fill_values : tuple, list of tuple
         specification of fill values for bad or missing table values
     fill_include_names : list
         List of names to include in fill_values.
     fill_exclude_names : list
         List of names to exclude from fill_values (applied after ``fill_include_names``)
-    fast_reader : bool or dict
+    fast_reader : bool, str or dict
```

Comment:
Oxford comma. :smile_cat: 
```suggestion
    fast_reader : bool, str, or dict
```